zetaclient/bitcoin_client.go

```diff
@@ -19,14 +19,14 @@ import (
 	"strconv"
 	"sync"
 	"sync/atomic"
-	"time"
 
 	"github.com/btcsuite/btcd/btcjson"
 	"github.com/btcsuite/btcd/rpcclient"
 	"github.com/btcsuite/btcutil"
 	"github.com/rs/zerolog"
 	"github.com/zeta-chain/zetacore/common"
 	"github.com/zeta-chain/zetacore/x/crosschain/types"
+	ostypes "github.com/zeta-chain/zetacore/x/observer/types"
```

Comment:
```suggestion
	crosschaintypes "github.com/zeta-chain/zetacore/x/crosschain/types"
	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
```
I think we should use [moduleName]types for the import name when not in the module for consistency throughout the code

Reply:
Agreed